### ISSUE TITLE
make_rss "Invalid template specified: Template not found: default.template (rss)"

### ISSUE THREAD

**Issue by emmercm** [Flexget/Flexget#901]
I've been wrestling with make_rss for hours now and I'm convinced something has changed to make it no longer work. Using an old config of mine that was working from October 2015 and running it through the latest PyPi flexget version (1.2.464) I am getting the error message in the title.

My config is complicated, but here is a simple one that produces the same error:
```
templates:
  global:
    make_rss:
      file: debug.rss

tasks:
  TV:
    rss: https://eztv.ag/ezrss.xml
    accept_all: yes
```

I've tried this with Python 2.7.11 on Windows 7, Python 2.7.10 through Cygwin on the same Windows 7 machine, and Python 2.7.9 on a Raspberry Pi 2 B. All using the same 1.2.464 PyPi release.

**Comment by emmercm** [comment]
It also looks like an old Trac ticket 2644 (http://flexget.com/ticket/2644) is potentially related and was never resolved, I am able to reproduce that issue with the above config and `template: '{{description}}'` added to `make_rss`.

**Comment by stevezau** [comment]
should be fixed now

**Comment by emmercm** [comment]
Confirmed fixed in 1.2.468, thank you.
